### PR TITLE
feat: Support for TFSec

### PR DESCRIPTION
**Pull request by jon-proietti-nutrien**
Add an integration for TFSec: https://github.com/liamg/tfsec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +116,13 @@ if they are present in `README.md`.
           - 'args=--enable-rule=terraform_documented_variables'
     ```
 
+## Notes about terraform_tfsec hooks
+
+1. `terraform_tfsec` will recurse all directories/modules.
+1. To ignore specific warnings, follow the convention from the
+[documentation](https://github.com/liamg/tfsec#ignoring-warnings).
```
Comment (antonbabenko) [inline README.md:123]: Please show sample configuration in notes where some warnings are ignored

File: README.md [hunk 2]
```diff
@@ -9,11 +9,13 @@
 * [`pre-commit`](https://pre-commit.com/#install)
 * [`terraform-docs`](https://github.com/segmentio/terraform-docs) required for `terraform_docs` hooks. `GNU awk` is required if using `terraform-docs` older than 0.8.0 with Terraform 0.12.
 * [`TFLint`](https://github.com/terraform-linters/tflint) required for `terraform_tflint` hook.
+* [`TFSec`](https://github.com/liamg/tfsec) required for `terraform_tsec` hook.
```
Comment (antonbabenko) [inline README.md:12]: terraform_tsec => terraform_tfsec

File: README.md (hunk 1 above)
Comment (jon-proietti-nutrien) [reply README.md:123]: Updated

**Comment by jon-proietti-nutrien** [comment]
Thanks @antonbabenko!

**Comment by antonbabenko** [comment]
You are welcome!

[v1.30.0](https://github.com/antonbabenko/pre-commit-terraform/releases/v1.30.0) has been just released.

**Pull request merged**